practice-one/src/practice-one.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="practice-one.css">
+        <title>practice-one</title>
+    </head>
+    <body>
+        <div class="nav">
+            <div class="frame">
+                <img src="../img/logo.png" class="logo" alt="">
+                <div class="open-enterprise">
+                    <span>Open Enterprise</span>
+                </div>
+            </div>
+            <div>
```

Comment:
Why need div here?

Reply:
To create a separate block that does not match the tags above